### DISCUSSION TITLE
Map Reveal Feature

### DISCUSSION THREAD

**Discussion by dexilan01** [Ceddini/CoreKeeperMapTool#18] [Feature Requests]
Can a feature be added to reveal to full map for the currently uploaded world file? The current one seems to be for a different world seed altogether

**Comment by Ceddini** [comment]
Sorry, but that's impossible. 
The map file the tool shows you is just an image (your minimap), so there's no way for me to reveal more. The map thing is also just an example on how a revealed map looks. The only real way to do it is in-game with mods
